com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -237,40 +237,41 @@ internal void __endSendClientRpc(ref FastBufferWriter bufferWriter, uint rpcMeth
         /// <summary>
         /// Gets if the object is the the personal clients player object
         /// </summary>
-        public bool IsLocalPlayer => NetworkObject.IsLocalPlayer;
+        public bool IsLocalPlayer => m_NetworkObject != null ? m_NetworkObject.IsLocalPlayer : false;
```

Comment:
Is this actually an improvement though?  If the NO is null, I think `IsLocalPlayer` should be undefined / access should crash as we actually don't really know.  If there's code later that wants to tolerate not having the NO set up yet, fine, then let that code do this check.  Also, since we call this frequently...

Reply:
I went ahead and made this a private set and just initialize it during the spawning/despawning stages.
It removes the processing overhead and is cleaner.